src/uwtools/file.py

```diff
@@ -36,10 +36,11 @@ def __init__(
         :param dry_run: Do not copy files
         :raises: UWConfigError if config fails validation.
         """
+        if dry_run:
+            dryrun()
```

Comment:
The dry-run option wasn't actually being used in the `file` mode. Now it is.

In the process of fixing this, though, I realized that API users may have a problem here: If a driver or other tool using `iotaa` is called with `dry_run=True`, `iotaa`' dry-run mode will be enabled and will not be subsequently disabled, even if another driver/tool is called with `dry_run=False`. I think an easy fix may be to just call `dryrun(enabled=dry_run)`, which will _always_ toggle `iotaa`'s dry-run mode to either `True` or `False`. I've added a reminder for myself to look into this, but thought I'd mention it.